atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -149,6 +156,48 @@ def get_buildroot(self, worker_metadatas):
 
         return buildroots
 
+    def get_logs(self):
+        """
+        Build list of log files
+
+        :return: list, of log files
+        """
+
+        logs = None
+        output = []
+
+        # Collect logs from server
+        try:
+            logs = self.osbs.get_orchestrator_build_logs(self.build_id)
+        except OsbsException as ex:
+            self.log.error("unable to get build logs: %r", ex)
+            return output
+        except TypeError:
+            # Older osbs-client has no get_orchestrator_build_logs
+            self.log.error("OSBS client does not support get_orchestrator_build_logs")
+            return output
+
+        platform_logs = {}
+        for entry in logs:
+            platform = entry.platform
+            if platform not in platform_logs:
+                filename = 'orchestrator' if platform is None else platform
+                platform_logs[platform] = NamedTemporaryFile(prefix="%s-%s" %
+                                                             (self.build_id, filename),
+                                                             suffix=".log", mode='wb')
+            else:
+                platform_logs.setdefault(platform, [])
```

Comment:
This looks wrong. This is saying that if `platform` is not a key in the `platform_logs` dict, it should be added with the value being an empty list.

But we already know it is a key in the dict, or we wouldn't get here.

Also why `[]`? The values are meant to be file-like objects, not lists.

Reply:
Bah. I'm not sure why I thought that snippet needed to stay in some form. Will just remove it.